client/components/TestRun/index.jsx

```diff
@@ -68,21 +68,26 @@ const TestRun = () => {
 
     const { runId: testPlanRunId, testPlanReportId } = params;
 
-    const { loading, data, error } = useQuery(
+    const { loading, data, error, startPolling, stopPolling } = useQuery(
         testPlanRunId ? TEST_RUN_PAGE_QUERY : TEST_RUN_PAGE_ANON_QUERY,
         {
             fetchPolicy: 'cache-and-network',
-            variables: { testPlanRunId, testPlanReportId }
+            variables: { testPlanRunId, testPlanReportId },
+            pollInterval
```

Comment:
I understand why this is a good idea but I wonder if the polling could be more targeted.  It feels extensive and triggers a re-render of most of the page. Fine with it for now though!

Reply:
Yeah - I was thinking about this and it would be a bit nicer to only poll the "status"es - I might give this a revision for this purpose

Reply:
I should be around on monday, but if not @stalgiag - I think we could merge this as is (unless others think we should wait) and I could do this part on a second pass